setup.py

```diff
@@ -31,14 +31,14 @@
 elif platform.system() == 'Darwin':
     extra_compile_args = FLAGS_CLANG + MACOS_FLAG
     extra_link_args = LFLAGS_CLANG + MACOS_FLAG
-elif platform.system() == 'FreeBSD':
+elif platform.system() == 'FreeBSD' or platform.system() == 'OpenBSD':
```

Comment:
```suggestion
elif platform.system() in {'FreeBSD', 'OpenBSD'}:
```

Reply:
That's obviously much better :) Thank you.